src/main/java/com/google/codeu/servlets/MarkerServlet.java

```diff
@@ -96,4 +114,33 @@ public void doDelete(HttpServletRequest request, HttpServletResponse response) t
       response.getOutputStream().println(json);
     }
   }
+  private String getFileUrl(HttpServletRequest request, String formInputElementName) {
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    Map<String, List<BlobKey>> blobs = blobstoreService.getUploads(request);
+    List<BlobKey> blobKeys = blobs.get(formInputElementName);
```

Comment:
Probably want to check if blobs.containsKey(formInputElementName), or do blobs.getOrDefault(formInputElementName, null) to avoid null pointer exception here

Reply:
Added the condition and if that is the case, return null